field/src/polynomial/mod.rs

```diff
@@ -64,7 +64,7 @@ impl<F: Field> PolynomialValues<F> {
     }
 
     pub fn degree_plus_one(&self) -> usize {
-        self.ifft().degree_plus_one()
+        self.clone().ifft().degree_plus_one()
```

Comment:
This might add a non-negligible cost, since we `trim` polynomials in a couple places which calls this. Though I think we could remove the `trim`s since we always know what degree a poly in our argument should have (unless it happens to have extra 0s by chance). So probably not worth worrying about. I think @wborgeaud was going to remove one of the `trim` calls

Reply:
The buffer was cloned before this change, so this PR doesn't introduce any _new_ inefficiencies. I'll make a new issue to look into the `trim`s.